media/main.js

```diff
@@ -89,6 +88,45 @@ function () {
 		vscode.setState( { oldBody: textToBase64, oldTitle : this.title, oldUri : passedUri });
 	}
 
+	function handleSingleRequirementCheck(element, isAvailable, requirementName) {
+		// add check for adoc div/p requirement label
+		if (element.tagName.toLowerCase() === 'div' && element.childNodes.length > 0) {
+			let list = element.getElementsByTagName('em');
+			if (list.length > 0) {
+				element = list[0];
+			}
+		}
+		
+		if (element) {
+			let green = "green";
```

Comment:
The variables are used in just a single place. Extracting them to variables instead of just assigning them directly in the following if else logic is not creating any benefit. I would either remove these variables fully or at least make them a const because they are not changed afterwards, but really removing them seems the better approach to me.